arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -12,6 +12,27 @@
 @author: Mark V Systems Limited
 (c) Copyright 2018 Mark V Systems Limited, All rights reserved.
 
+GUI operation
+
+   install plugin validate/ESEF and optionally applicable taxonomy packages
+   
+   Under tools->formula add parameters eps_threshold and optionally authority
+
+Command line operation:
+
+   arelleCmdLine.exe --plugins validate/ESEF --packages {my-package-directory}/esef_taxonomy_2019.zip
+     --disclosureSystem esef -v -f {my-report-package-zip-file}
+   Adding checks for formulas not automatically included:
+     --parameters "eps_threshold=.01"
+   Dimensional validations required by some auditors may require
+    --import http://www.esma.europa.eu/taxonomy/2020-03-16/esef_all-for.xml
+    and likely --skipLoading *esef_all-cal.xml 
+    because the esef_all-cal.xml calculations are reported to be problematical for some filings
+
+Authority specific validations are enabled by formula parameter authority, e.g. for Danemark or UKSEF and eps_threshold specify:
+     --parameters "eps_threshold=.01,authority=DK"
+     --parameters "eps_threshold=.01,authority=UK"
```

Comment:
Just wondering what your thought process was on adding the authority to the parameters and not just making a new disclosure system that includes a preset authority parameter?

Reply:
This is how EDGAR validation works, there are a lot of filing parameters passed in using the json filename option which overlaps with formula parameters - see the validate/EFM/__init__ header.  Would like to have a common style.